config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/Service.java

```diff
@@ -203,11 +204,19 @@ public String getName() {
 
             @Override
             public String getId() {
+                AnnotatedEntity annotatedEntity = getAnnotatedEntity();
```

Comment:
do we have to consider bundle-hints annotations only for shared entities or all entities ? earlier we were considering this only for shared entities.

Reply:
we have to consider it for all code entities and some env entities like trusted cert, generic entity etc

Reply:
For Policy and Encass, we are using this bundle-hints id and guid only in case of Shared entity

Reply:
so, we do not have to consider it for service?